packages/basti/src/aws/elasticache/get-elasticache-memcached-clusters.ts

```diff
@@ -0,0 +1,75 @@
+import { DescribeCacheClustersCommand } from '@aws-sdk/client-elasticache';
+
+import {
+  parseMemcachedCacheClusteWithSpecificNoderResponse,
+  parseMemcachedCacheClusterResponse,
+} from './parse-elasticache-memcached-response.js';
+import { elasticacheClient } from './elasticache-client.js';
+
+import type { CacheCluster } from '@aws-sdk/client-elasticache';
+import type { AwsElasticacheMemcachedCluster } from './elasticache-types.js';
+
+export interface getCacheClusterInput {
```

Comment:
Please, uppercase this type. Maybe we have to add an eslint rule for casing? 🤔

Reply:
great idea